package.json

```diff
@@ -31,14 +31,17 @@
     "ts-check": "npx tsc --noEmit",
     "clean": "cd example && npx react-native-clean-project --keep-node-modules --remove-iOS-build --remove-iOS-pods --remove-android-build --clean-android-project --keep-brew --keep-pods && \\rm -fr ios/Pods",
     "dev-sync": "yarn build && cp -r package.json *podspec lib windows android ios src jest example/node_modules/react-native-device-info/",
+    "dev-sync-windows": "for %x in (lib windows android src jest) do xcopy \"%x\" .\\example\\node_modules\\react-native-device-info\\ \/S \/Y",
     "lint": "npx eslint ./ --ignore-pattern example --ignore-pattern node_modules --fix --quiet",
     "shipit": "yarn clean && semantic-release",
     "example:install": "yarn && cd example && yarn && cd .. && yarn dev-sync",
+    "example:install-windows": "yarn && cd example && yarn && cd .. && yarn dev-sync-windows",
     "example:install:pods": "cd example/ios && pod install",
     "example:android": "cd example && yarn android",
     "example:android:build": "cd example/android && ./gradlew assembleDebug",
     "example:ios": "cd example && yarn ios",
     "example:ios:build": "cd example/ios && xcodebuild CC=clang CPLUSPLUS=clang++ LD=clang LDPLUSPLUS=clang++ -workspace example.xcworkspace -configuration Debug -scheme example -sdk iphonesimulator -arch x86_64 ONLY_ACTIVE_ARCH=YES | xcpretty",
+    "example:windows": "cd example && yarn windows",
```

Comment:
This file has the only useful changes in this commit, but the other files were perturbed by the windows build and it seemed best to include them
The build did work successfully locally on windows! And it works in CI too (which is VS16 toolchain, older than my local one) which proves we work on a pretty wide windows toolchain skew right now